build.sbt

```diff
@@ -17,25 +19,13 @@ lazy val microservice = Project(AppDependencies.appName, file("."))
   .settings(publishingSettings: _*)
   .configs(IntegrationTest)
   .settings(
-    Keys.fork in IntegrationTest := false,
+    IntegrationTest / Keys.fork := false,
     Defaults.itSettings,
     IntegrationTest / unmanagedSourceDirectories += baseDirectory(_ / "it").value,
-    IntegrationTest / parallelExecution := false,
-    IntegrationTest / testGrouping := oneForkedJvmPerTest((IntegrationTest / definedTests).value),
```

Comment:
Any reason to remove this line?

Reply:
Import changed, just didn't re-add it, will add it back in with the new import :)